dashboard/legacy/middleware/tables_api.rb

```diff
@@ -22,6 +24,7 @@ class TablesApi < Sinatra::Base
   # Exports a csv file from a table where the first row is the column names
   # and additional rows are the column values.
   #
+  # TODO: post-firebase-cleanup, remove this method at the least, probably remove whole file: #56994
   get %r{/v3/export-firebase-tables/([^/]+)/([^/]+)$} do |channel_id, table_name|
```

Comment:
For DatablockStorage, this is instead implemented as a regular route on DatablockStorageController. After fully migrating, we can probably delete table_api.rb.